platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -257,106 +227,101 @@ public void testConstructorWithNullConfigurationFileExtension() {
                 null);
     }
 
-    @Test
-    public void testInitDoesNotCreateExistingProcessedAndFailedAndExportedDirectories()
-            throws IOException {
-        when(processedDirectory.exists()).thenReturn(true);
-        when(processedDirectory.isDirectory()).thenReturn(true);
-        when(failedDirectory.exists()).thenReturn(true);
-        when(failedDirectory.isDirectory()).thenReturn(true);
+    @Test(expected = IOException.class)
+    public void testInitDoesNotCreateExistingProcessedAndFailedDirectories() throws IOException {
```

Comment:
Just realized that this test is probably not needed anymore. It was used to make sure that both directories were created even when they existed but since we now use `FileUtils.forceMkdir()`, the code path is the same as if the directories didn't exist.

Keeping the other comments on that method though as they apply to other tests :-)